lib/beaker/hypervisor/openstack.rb

```diff
@@ -63,6 +63,11 @@ def initialize(openstack_hosts, options)
         raise "Unable to create OpenStack Network instance (api_key: #{@options[:openstack_api_key]}, username: #{@options[:openstack_username]}, auth_url: #{@options[:openstack_auth_url]}, tenant: #{@options[:openstack_tenant]})"
       end
 
+      # Validate openstack_volume_support setting value, reset to boolean if passed via ENV value string
+      @options[:openstack_volume_support] = true  if @options[:openstack_volume_support].to_s.match(/\btrue\b/i)
+      @options[:openstack_volume_support] = false if @options[:openstack_volume_support].to_s.match(/\bfalse\b/i)
+      [true,false].include? @options[:openstack_volume_support] or raise "Invalid openstack_volume_support setting, current: @options[:openstack_volume_support]"
```

Comment:
@ipcrm what advantages does this approach have over a much simpler one like below?
```ruby
@options[:openstack_volume_support] = @options[:openstack_volume_support].to_s == 'true'
```